src/lib.rs

```diff
@@ -334,7 +334,7 @@ fn set(
 }
 
 #[macro_export]
-#[cfg(not(feature = "disabled"))]
```

Comment:
How about changing the name to "enable_fail_point"? Then the first condition can be:
#[cfg(feature = "enable_fail_point")]
And the second the condition:
+#[cfg(not(feature = "enable_fail_point"))]

Reply:
I prefer enabling fail points by default. When people include a crate as a dependency, it's supposed to work without any additional configuration. An extra (optional) features is just a convenient way to disable all fail points for people who don't want to put `#[cfg(features)]` around.

Reply:
How about "no_fail", I rarely see dormancy before.